solidity/random-beacon/contracts/Reimbursable.sol

```diff
@@ -28,9 +27,13 @@ abstract contract Reimbursable is Ownable {
         reimbursementPool.refund(gasStart - gasleft(), receiver);
     }
 
+    modifier onlyReimbursableAdmin() virtual {
```

Comment:
Why not `onlyReimbursableGovernance`? Are we expecting to have a different "admin" other than governance?

Reply:
I am pretty sure the assumption was that the contract inheriting from `Reimbursable` does not need to inherit from `Governable`. I will leave this thread open so that @nkuba can clarify.